### PR TITLE
v2 self-hosted changes

### PR DESCRIPTION
**Pull request by ruslandoga**
This PR updates the self-hosted docs for `v2` release (together with https://github.com/plausible/docs/pull/396).

### CODE REVIEW

File: docs/self-hosting-configuration.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ Following are the variables that can be used to configure the availability of th
...
 | LISTEN_IP            | 0.0.0.0 | The IP address on which the server is listening. `0.0.0.0` means all interfaces, `127.0.0.1` means localhost. Also see the related section **Erlang platform ports** below.                         |
 | SECRET_KEY_BASE      | --      | An internal secret key used by [Phoenix Framework](https://www.phoenixframework.org/). Follow the [instructions](https://hexdocs.pm/phoenix/Mix.Tasks.Phx.Gen.Secret.html#content) or use `openssl rand -hex 64` to generate one. |
-| DISABLE_REGISTRATION | false   | Restricts registration of new users. Possible values are `true` (full restriction), `false` (no restriction), and `invite_only` (only the invited users can register).                                                                                                                            |
+| DISABLE_REGISTRATION | true    | Restricts registration of new users. Possible values are `true` (full restriction), `false` (no restriction), and `invite_only` (only the invited users can register).                                                                                                                            |
```
Comment (ruslandoga) [inline docs/self-hosting-configuration.md:21]:
```suggestion
| DISABLE_REGISTRATION | invite_only | Restricts registration of new users. Possible values are `true` (full restriction), `false` (no restriction), and `invite_only` (only the invited users can register).                                                                                                                            |
```

**Comment by ruslandoga** [comment]
@metmarkosaric 👋

I think this is ready :)

File: docs/self-hosting.md [hunk 2]
```diff
@@ -107,7 +107,8 @@ The Plausible server itself does not perform SSL termination. It only runs on un
 ## Updating Plausible
 
 Plausible is updated regularly, but it's up to you to apply these updates on your server.
-By virtue of using Docker, these updates are safe and easy to apply.
+You may refer to the discussions on [the releases page](https://github.com/plausible/analytics/releases) for specific instructions.
+The typical steps for handling minor version updates are as follows:
```
Comment (ruslandoga) [inline docs/self-hosting.md:111]: https://github.com/plausible/analytics/discussions/3011#discussioncomment-6459361

**Comment by metmarkosaric** [comment]
thanks @ruslandoga! merged both!

**Comment by ruslandoga** [comment]
Thank you!

**Pull request merged**